JS/Lesson15/esra/script.js

```diff
@@ -113,52 +113,58 @@ console.log('------------------------');
 // 16. Extract the price and currency from a string below using "slice" method and print it to console.
 const planPriceString = 'Premium plan - 9.99 USD/month.'; // Should return "9.99 USD"
 
-console.log('Exercise16: ' /* Result here */);
+console.log('Exercise16: ', planPriceString.slice(14, 23));
```

Comment:
Bu sekilde dene, calistirip, kodu kontrol et
```suggestion
console.log('Exercise16: ', planPriceString.slice(planPriceString.indexOf('- 9'),planPriceString.indexOf('/month')));
```